lib/parse-comment.js

```diff
@@ -35,6 +35,8 @@ const validContributionTypes = [
   "tutorial",
   "usertesting",
   "video",
+  "research",
+  "promotion",
```

Comment:
TODO: Add an emoji key for `promotion` in https://github.com/all-contributors/cli

Reply:
U mean adding emoji in docs here 
> https://github.com/all-contributors/all-contributors/blob/master/docs/emoji-key.md